src/main/java/sirius/biz/jobs/params/SelectStringParameter.java

```diff
@@ -64,6 +64,9 @@ protected String checkAndTransformValue(Value input) {
 
     @Override
     protected Optional<String> resolveFromString(@Nonnull Value input) {
+        if (input.isEmptyString()) {
+            return Optional.empty();
+        }
         return Optional.of(input.asString());
```

Comment:
```suggestion
        return Optional.of(input.asString()).filter(Strings::isFilled);
```
Ist ein bisschen ideomatischer (evtl auch `ofNullable`)

Reply:
Making a pre-check would avoid running an unnecessary `Value.asString`, which validates the underlying object against several conditions defined by NLS.toMachineString.

Reply:
```suggestion
        return input.asOptionalString();
```
Checked again: What you want is this. It does exactly what you did yourself, as far as i can tell

Reply:
It does! Thx for checking.